tutorials/Introduction.ipynb

```diff
@@ -77,21 +77,22 @@
       "execution_count": null,
       "id": "d2f69f4e",
       "metadata": {
-        "colab": {
-          "base_uri": "https://localhost:8080/"
-        },
-        "id": "bZLi0pmmSKZW",
-        "outputId": "a24f128d-82d1-4623-d36e-074dd72aeeb9"
+        "id": "d2f69f4e",
+        "vscode": {
+          "languageId": "python"
+        }
       },
       "outputs": [],
       "source": [
-        "!pip install pyclarify"
+        "!pip install -U pyclarify"
```

Comment:
What is the use of -U ?

Reply:
Its shorthand for `--upgrade`, I used it when testing, will remove :)